prawns/governance/src/processors/khalaProcessor.ts

```diff
@@ -6,10 +6,16 @@ import councilProposedHandler from '../handlers/council.Proposed.event';
 import technicalCommitteeProposedHandler from '../handlers/technicalCommittee.Proposed.event';
 import democracySecondHandler from '../handlers/democracy.second.extrinsic';
 import electionVoteHandler from '../handlers/phragmenElection.vote.extrinsic';
+import bountiesBountyProposedHandler from "../handlers/bounties.BountyProposed.event";
+import treasuryProposedHandler from "../handlers/treasury.Proposed.event";
```

Comment:
I think the files you created should be camel cased after the second dot.

Reply:
Sure, I thought these names had to reflect the casing of the events and extrinsics in the docs based on ones already there but I prefer this too.

Reply:
```suggestion
import bountiesBountyProposedHandler from "../handlers/bounties.bountyProposed.event";
import treasuryProposedHandler from "../handlers/treasury.proposed.event";
```

Nitpicking: but you forgot to remove capital letters here! 😄 All works, I know, that's why I'm approving as this is optional 👍